src/javascript/app/pages/trade/barriers.js

```diff
@@ -160,14 +161,75 @@ const Barriers = (() => {
         Defaults.remove('barrier', 'barrier_high', 'barrier_low');
     };
 
-    const validateBarrier = () => {
+    const showError = (barrier) => {
+        if (!barrier.classList.contains('error-field')) {
+            barrier.classList.add('error-field');
+        }
+        const error_node = barrier.parentNode.lastElementChild.firstElementChild;
```

Comment:
this is too path dependent, can we use some class identifier?

Reply:
In that way, we have to pass extra two params to `hideError` `showError`. And I wanted to avoid it.

There are two ways to make this better:
1) (Your suggestion) pass extra two params 
2) Or we can dynamically add or remove DOM. 
- Append Error DOM dynamically.`const  newDom = createElementDOM('<div>').className='hint'; 

Which way do you think it's better?

Reply:
Oh we can get the DOM from hideError function. Uhmm

Reply:
let's keep it as is for now

Reply:
wait, I got confused. this comment is about using `.parentNode.lastElementChild.firstElementChild` instead of some common class like in parent, look for child with class error-message or something like that, to avoid depending on it being the first child of the last child

Reply:
No, actually your first suggestion was absolutely correct and better :) Thanks